CSE115A/rideshare-project
# Google map

Problem
=======
Adding the map and getting it to display

[Link to Ticket](https://trello.com/c/PQf8CFzX/92-google-maps-front-end-api-task-1)

Solution
========
Added a Map component with hard-coded lat and long values, which calls the Google Maps API and displays a Google Map. A new API key was added to the Firestore, and Github Secrets. This API is not called through any Firebase function(s). We may need to host this call on a firebase function just like we did for getting the Uber and Lyft prices.

Change summary:
---------------
* start of google map box - fe4275
* added rest of map code - 2bf773
* got rid of api key - b20149
* fixed maps - 3b6746
* deleted API key at top of landingpage index - 1498b2

Steps to Verify:
----------------
1. Run `yarn start` in a terminal
2. Go to the landing page
3. A Google Map will appear in the bottom of the page with a pinned location

Screenshots (optional):
-----------------------
![image](https://user-images.githubusercontent.com/37026447/99717353-26223300-2a5e-11eb-9b2f-83fb8cfbfc9f.png)
